src/diffusers/modeling_flax_utils.py

```diff
@@ -45,7 +45,7 @@ class FlaxModelMixin:
     """
     config_name = CONFIG_NAME
     _automatically_saved_args = ["_diffusers_version", "_class_name", "_name_or_path"]
-    _flax_internal_args = ["name", "parent"]
+    _flax_internal_args = ["name", "parent", "dtype"]
```

Comment:
did you receive a warning while initializing?

Reply:
> did you receive a warning while initializing?

Not about `dtype`. In the case of the VAE, for example, I did the following tests:

- Loaded from the current configuration in `fusing`. Then I got this warning:

```
The config attributes {'architectures': ['encoderKL'], 'double_z': True, 'transformers_version': '4.21.0.dev0'} were passed to FlaxAutoencoderKL, but are not expected and will be ignored. Please verify your config.json configuration file.
```

- Saved the configuration.
- Saved the pretrained weights.
- Loaded from the saved pretrained weights. No warning was emitted here.